tests/kvmtest.sh

```diff
@@ -250,6 +250,7 @@ test_t1_lag() {
     route/test_default_route.py \
     scp/test_scp_copy.py \
     test_interfaces.py"
+    tests="fips/test_fips.py"
```

Comment:
To enable fips testing in PR testing, need to add it into file `.azure-pipelines/pr_test_scripts.yaml`. The `kvmtest.sh` file is being deprecated.